demo/nexmark/nexmark.sql

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE person (
+    id BIGINT,
+    name VARCHAR,
+    emailAddress VARCHAR,
+    creditCard VARCHAR,
+    city VARCHAR,
+    state VARCHAR,
+    date_time TIMESTAMP(3), -- NOT NULL LATENESS INTERVAL 4 SECONDS,
+    extra  VARCHAR
+);
+CREATE TABLE auction (
+    id  BIGINT,
+    itemName  VARCHAR,
+    description  VARCHAR,
+    initialBid  BIGINT,
+    reserve  BIGINT,
+    date_time  TIMESTAMP(3), -- NOT NULL LATENESS INTERVAL 4 SECONDS,
+    expires  TIMESTAMP(3),
+    seller  BIGINT,
+    category  BIGINT,
+    extra  VARCHAR
+);
+CREATE TABLE bid (
+    auction  BIGINT,
+    bidder  BIGINT,
+    price  BIGINT,
+    channel  VARCHAR,
+    url  VARCHAR,
+    date_time TIMESTAMP(3), -- NOT NULL LATENESS INTERVAL 4 SECONDS,
+    extra  VARCHAR
+);
+-- -------------------------------------------------------------------------------------------------
+-- Query 9: Winning Bids (Not in original suite)
+-- -------------------------------------------------------------------------------------------------
+-- Find the winning bid for each auction.
+-- -------------------------------------------------------------------------------------------------
+
+CREATE VIEW q9 AS
+SELECT
+    id, itemName, description, initialBid, reserve, date_time, expires, seller, category, extra,
+    auction, bidder, price, bid_dateTime, bid_extra
+FROM (
+   SELECT A.*, B.auction, B.bidder, B.price, B.date_time AS bid_dateTime, B.extra AS bid_extra,
+     ROW_NUMBER() OVER (PARTITION BY A.id ORDER BY B.price DESC, B.date_time ASC) AS rownum
+   FROM auction A, bid B
+   WHERE A.id = B.auction AND B.date_time BETWEEN A.date_time AND A.expires
+)
+WHERE rownum <= 1;
+
+-- -------------------------------------------------------------------------------------------------
+-- Query 10: Log to File System (Not in original suite)
```

Comment:
clearly, this doesn't involve a filesystem... 
so perhaps some of these comments could be modified

Reply:
This comment comes from your NexmarkTest.java.

Reply:
It was copied by Michael from the Java flink code

Reply:
I deleted all the SQL comments. They aren't helpful here.